qctrlopencontrols/cirq/circuit.py

```diff
@@ -0,0 +1,225 @@
+# Copyright 2019 Q-CTRL Pty Ltd & Q-CTRL Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+============
+cirq.circuit
+============
+"""
+
+import numpy as np
+
+import cirq
+
+from qctrlopencontrols.dynamic_decoupling_sequences import DynamicDecouplingSequence
+from qctrlopencontrols.exceptions import ArgumentsValueError
+
+from qctrlopencontrols.qiskit import (
```

Comment:
cirq should not import anything from qiskit. Could you pull these constants into global and put the method in a module where it would be more appropriate.